src/Libraries/RevitNodes/Schedules/SchedulableField.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+using RevitServices.Persistence;
+
+namespace Revit.Schedules
+{
+    /// <summary>
+    ///     Revit Schedule Field
+    /// </summary>
+    public class SchedulableField
+    {
+
+        #region Internal Properties
+
+        /// <summary>
+        ///     Internal reference to the Revit Element
+        /// </summary>
+        internal Autodesk.Revit.DB.SchedulableField InternalSchedulableField
+        {
+            get; set;
+        }
+
+        /// <summary>
+        ///     Internal constructor. Used by the public static constructor to construct an instance of the SchedulableField class.
+        /// </summary>
+        /// <param name="internalSchedulableField"></param>
+        internal SchedulableField(Autodesk.Revit.DB.SchedulableField internalSchedulableField)
+        {
+            this.InternalSchedulableField = internalSchedulableField;
+        }
+
+        #endregion
+
+        #region Public properties
+
+        /// <summary>
+        ///     Name
+        /// </summary>
+        public string Name
+        {
+            get
+            {
+                var doc = DocumentManager.Instance.CurrentDBDocument;
+                return this.InternalSchedulableField.GetName(doc);
+            }
+        }
+
+        public override string ToString()
+        {
+            return this.Name;
```

Comment:
not for a Key Schedule. Key Schedule by default will always have at least one schedulable field which is named Key Name. That's one that is user facing, but there are always system ones that are visible in API but not in the UI. I am going to take an educate guess and say the same is true for all other schedule types, but it remains to be seen as we get there.